ykcompile/src/lib.rs

```diff
@@ -717,4 +877,132 @@ mod tests {
         assert_eq!(expect, 7);
         assert_eq!(expect, got);
     }
+
+    fn many_locals() -> u8 {
+        let _a = 1;
+        let _b = 2;
+        let _c = 3;
+        let _d = 4;
+        let _e = 5;
+        let _f = 6;
+        let h = 7;
+        let _g = true;
+        h
+    }
+
+    #[test]
+    fn test_spilling_simple() {
+        let th = start_tracing(Some(TracingKind::HardwareTracing));
+        many_locals();
+        let sir_trace = th.stop_tracing().unwrap();
+        let tir_trace = TirTrace::new(&*sir_trace).unwrap();
+        let (ct, spills) = TraceCompiler::test_compile(tir_trace);
+        assert_eq!(ct.execute(), 7);
+        // Changing the registers available to the register allocator affects the amount of spills,
```

Comment:
This is a nitpick, but I *think* `s/amount/number/` as `amount` implies an uncountable unit, like water. Might be wrong. Just my gut feeling.

Reply:
Fixed in 11b078a